src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -70,6 +72,16 @@ internal AssemblyScanner(Assembly assemblyToScan)
             ThrowExceptions = true;
         }
 
+        void DetermineIfNestedDirectoriesShouldBeExcluded()
+        {
+            bool scanNestedDirectories;
+            var appSetting = ConfigurationManager.AppSettings["NServiceBus/AssemblyScanning/ScanNestedDirectories"];
```

Comment:
@andreasohlund I'll have to look into this. It was changed to rely on code only config...